mypy_django_plugin/transformers/querysets.py

```diff
@@ -784,7 +784,7 @@ def _validate_bulk_update_field(
         ctx.api.fail(f'"{method}()" can only be used with concrete fields. Got "{field_name}"', ctx.context)
         return False
 
-    all_pk_fields = set(opts.pk_fields)
+    all_pk_fields = set(getattr(opts, "pk_fields", [opts.pk]))
     for parent in opts.all_parents:
         all_pk_fields.update(parent._meta.pk_fields)
```

Comment:
We probably need to do the same here ?

Reply:
🤦🏻 Yes. Sorry - rushed this as I don't have much time right now. Pushed an update.